front-end/src/common/stats-generators/pings/TotalEnemyMissingPingStatGenerator.ts

```diff
@@ -0,0 +1,16 @@
+import {BaseStatGenerator} from "../BaseStatsGenerator";
+import PlayerStatModel from "../../../../../Common/models/playerstat.model";
+
+export class TotalEnemyMissingPingStatGenerator extends BaseStatGenerator {
+    getStatTitle(): string {
+        return "TEMP"
```

Comment:
I know this stands for Total Enemy Missing Pings, but when I first saw this I did a double take